### DISCUSSION TITLE
xz vulnerability.....is the wyzecam vulnerable

### DISCUSSION THREAD

**Discussion by sideup66** [gtxaspec/wz_mini_hacks#754] [General]
Kinda a longshot, but recently this vulnerability broke: 
https://arstechnica.com/security/2024/04/what-we-know-about-the-xz-utils-backdoor-that-almost-infected-the-world/

It looks like a pretty ugly exploit. Although wz_mini is a pretty stripped down system, it DOES have hooks involving ssh and frankly anything using lzma compression if the lib is new enough. Does the wz_mini system us these tools and if so what version? seems like 4.6.x is affected, though debian stable and redhat run older versions which right now are not seen as dangerous. Kinda curious to this

**Comment by gtxaspec** [comment]
Looks like only amd64 and systemd is targeted.  Embedded is safe from this for now lol
